VTEX - Budgets API.json

```diff
@@ -0,0 +1,5235 @@
+          "autoResetOnPeriodEnd",
+          "carryOverBalance"
+        ],
+        "properties": {
+          "startDate": {
+            "type": "string",
+            "format": "date-time",
+            "description": "Start date of the budget cycle (ISO format).",
+            "example": "2025-04-01T00:00:00.0000000Z"
+          },
+          "endDate": {
+            "type": "string",
+            "format": "date-time",
+            "description": "End date of the budget cycle (ISO format).",
+            "example": "2025-04-08T00:00:00.0000000Z"
+          },
+          "autoResetOnPeriodEnd": {
+            "type": "boolean",
+            "description": "Enable (`true`) or disable (`false`) the autoreset period.",
+            "example": true
+          },
+          "carryOverBalance": {
+            "type": "boolean",
+            "description": "Defines wether to carry over balance (`true`) or not (`false`).",
+            "example": true
+          }
+        }
+      },
+      "CycleConfigurationResponse": {
+        "type": "object",
+        "description": "Configuration for budget cycle in response.",
+        "properties": {
+          "startDate": {
+            "type": "string",
+            "format": "date-time",
+            "description": "Start date of the budget cycle (ISO format)."
+          },
+          "endDate": {
+            "type": "string",
+            "format": "date-time",
+            "description": "End date of the budget cycle (ISO format)."
+          },
+          "autoResetOnPeriodEnd": {
+            "type": "boolean",
+            "description": "Condition if the period will auto reset when ended."
+          },
+          "carryOverBalance": {
+            "type": "boolean",
+            "description": "Condition if the remain balance will be transferred to the next cycle."
+          },
+          "nextRenewal": {
+            "type": "string",
+            "format": "date-time",
+            "description": "Next cycle renewal date."
+          }
+        }
+      },
+      "ResetBudgetRequest": {
+        "type": "object",
+        "description": "Request body for resetting budget usage.",
+        "required": [
+          "origin",
+          "reference",
+          "amount",
+          "type",
+          "requestedBy"
+        ],
+        "properties": {
+          "origin": {
+            "type": "string",
+            "description": "Describes the origin or reason for the adjustment."
+          },
+          "reference": {
+            "type": "string",
+            "description": "A reference ID tied to the transaction (e.g., order ID)."
+          },
+          "amount": {
+            "type": "number",
+            "description": "The amount of budget used for this allocation. When set to `0`, your budget will correspond to USD 0."
+          },
+          "type": {
+            "type": "string",
+            "description": "Operation type, which can be `Credit` or `Debit`."
```

Comment:
Check to confirm if it shouldn't be UPPER case (CREDIT and DEBIT) or lower case.